editor/src/core/workers/parser-printer/parser-printer-test-utils.ts

```diff
@@ -484,6 +493,36 @@ export function topLevelElementArbitrary(): Arbitrary<TopLevelElement> {
   )
 }
 
+export function exportDetailNamedArbitrary(
+  possibleNames: Array<string>,
+): Arbitrary<ExportDetailNamed> {
+  return FastCheck.constantFrom(...possibleNames).map(exportDetailNamed)
+}
+
+export function exportDetailModifierArbitrary(): Arbitrary<ExportDetailModifier> {
+  return lowercaseStringArbitrary().map(exportDetailModifier)
+}
+
+export function exportDetailArbitrary(possibleNames: Array<string>): Arbitrary<ExportDetail> {
+  return FastCheck.oneof<ExportDetail>(
+    exportDetailNamedArbitrary(possibleNames),
+    exportDetailModifierArbitrary(),
+  )
+}
+
+export function exportsDetailArbitrary(possibleNames: Array<string>): Arbitrary<ExportsDetail> {
+  if (possibleNames.length === 0) {
+    return FastCheck.constant(EmptyExportsDetail)
+  } else {
+    return FastCheck.tuple(
+      FastCheck.constant(null), //FastCheck.option(exportDetailNamedArbitrary(possibleNames)),
```

Comment:
What's with the commented code here?

Reply:
Ah yes, this is something I need to follow up on, it relates to `export default Thing = (props) => {...}` being invalid.